product_docs/docs/pgd/5/raft/02_raft_subgroups_and_pgd_cli.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Working with Raft subgroups and PGD CLI"
+---
+
+
+You can view the status of your nodes and subgroups with the [pgd](../cli/index) cli command. The examples here assume a cluster as configured by in [Creating Raft subgroups with TPA](raft_and_tpa).
+
+
+## Viewing nodes with pgd
+
+The pgd command's `show-nodes`.
+```shell
+$pgd show-nodes
+__OUTPUT__
+Node     Node ID    Node Group Type Current State Target State Status Seq ID 
+----     -------    ---------- ---- ------------- ------------ ------ ------ 
+east1    916860695  us_east    data ACTIVE        ACTIVE       Up     1      
+east2    2241077170 us_east    data ACTIVE        ACTIVE       Up     2      
+east3    1093023575 us_east    data ACTIVE        ACTIVE       Up     3      
+west1    1668243030 us_west    data ACTIVE        ACTIVE       Up     4      
+west2    2311995928 us_west    data ACTIVE        ACTIVE       Up     5      
+west3    4162758468 us_west    data ACTIVE        ACTIVE       Up     6      
+```
+
+## Viewing groups (and subgroups) with pgd
+
+To show the groups in a PGD deployment, along with their names and attributes, use the PGD cli command `show-groups.`
+
+```
+$pgd show-groups
```

Comment:
PJ,
Could you please change top group name to "pgdgroup" instead of "bdrgroup"? Can you also scan the doc if it needs to be fixed somewhere else in this doc?
Thanks!

Reply:
Fixed.